deploy/crds/app_v1alpha1_kogitoapp_crd-0.4.0.yaml

```diff
@@ -85,6 +85,45 @@ spec:
                     be compiled to run on native mode when Runtime is quarkus. See:
                     https://www.graalvm.org/docs/reference-manual/aot-compilation/'
                   type: boolean
+                resources:
+                  description: Resources for build pods. Default limits are 1GB RAM/0.5
+                    cpu on jvm and 2GB RAM/0.8 cpu for native builds.
```

Comment:
```suggestion
                    cpu on jvm and 4GB RAM/1 cpu for native builds.
```

Reply:
@sutaakar thanks! I already changed in my local git. Will push this plus the env vars change soon